### ISSUE TITLE
Estimate song length; seeking

### ISSUE THREAD

**Issue by madelynnblue** [korandiz/mpa#2]
I've been using your package with great success to play and stream mp3s. I'd now like to be able to add functionality to move around in a non-streamed mp3 and know it's length. Are these APIs that can be reasonably added?

**Comment by korandiz** [comment]
Yes, they can be added, but it isn't trivial, and it requires constructing a seek table. I'll look into it soon.

**Comment by korandiz** [comment]
Please let me know if this package solves your problem: https://github.com/korandiz/mpseek/
